### DISCUSSION TITLE
macos montery - macbook m1 - router keeps disconnecting

### DISCUSSION THREAD

**Discussion by mrhhats** [Ylianst/MeshCentral#4758] [General]
Hi there! running a macbook pro m1 with macos montery. Running router 0.0.3 and I start a RDP tunnel to a machine at another location. Tunnel works for around 2-3 mins and then the mapping just disappears from the gui and my tunnel drops.

Anyone else experience the same?

Thanks by the way for this amazing tool!

**Comment by silversword411** [comment]
2-3mins sounds like websockets dying.

Reverse proxy, firewall, connection timeouts somewhere.

**Comment by mrhhats** [comment]
Thank you @silversword411 - that was it! I adjusted the values in my nginx reverse proxy config and its now stable! cheers.
